UKHomeOffice/cerberus-service
# COP-9874: recipient detail layout fix

## Description
TIS recipient detail form layout fix

## To Test
Claim and issue a task on Roro, fill out the mandatory fields and when you get to the recipient details, then you see this
![image](https://user-images.githubusercontent.com/1534608/187597693-47a032d8-98e6-443b-add7-c9a92cf1c657.png)


![image](https://user-images.githubusercontent.com/1534608/187597743-2e7d1f56-a806-49ed-a693-4d7097663fcb.png)


## Developer Checklist

\* Required

- [*] Up-to-date with main branch? \*

- [*] Does it have tests?

- [*] Pull request URL linked to JIRA ticket? \*
